appmgr/src/install/mod.rs

```diff
@@ -126,7 +126,7 @@ pub async fn install(
     match pde.take() {
         Some(PackageDataEntry::Installed {
             installed,
-            manifest,
+            manifest: _manifest,
```

Comment:
this is to quiet a warning. I made changes in every file I touched for other reasons to take warnings to 0 in those modules.